traversal/planner/GraphPlanner.java

```diff
@@ -410,47 +385,34 @@ public String toString() {
         return str.toString();
     }
 
-    private class Initialiser {
```

Comment:
Initialiser acted as a method: the only use of it was `new Initialiser().execute()`, then the initialiser state was immediately discarded. It is explicitly a method now.